classes/command/class-shortcode-to-gutenberg.php

```diff
@@ -32,15 +32,13 @@ public function init() {
 				'shortcake_columns',
 				'shortcake_carousel_header',
 				'shortcake_cookies',
-				'shortcake_counter',
```

Comment:
I removed this one here too because it was forgotten when converting Counter to WYSIWYG 😬